src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,26 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double dozingWidth;
```

Comment:
The field 'dozingWidth' should have an access modifier to encapsulate the field properly. Consider using 'private' to keep it hidden from external access.